src/libraries/Native/Unix/System.Native/pal_networkstatistics.c

```diff
@@ -228,20 +228,26 @@ int32_t SystemNative_GetIcmpv4GlobalStatistics(Icmpv4GlobalStatistics* retStats)
 
 int32_t SystemNative_GetIcmpv6GlobalStatistics(Icmpv6GlobalStatistics* retStats)
 {
+    // macOS 11.0 added new member to icmp6stat. To make it binary compatible we simply allocate
+    // little bit more and that is ok on older systems. This will make it more resilient to additions in future.
+    // This avoid runtime detection and dynamic allocation. And we do not care about the new value.
+
+    #define ICMP6_STAT_PAD 32
```

Comment:
It's weird to define a macro inside a function and not undef it before leaving the function; A const should work just as well here and avoid that.